redis.go

```diff
@@ -130,3 +133,26 @@ func redisSentinelConnection(settings RedisSettings) (redis.Conn, error) {
 
 	return pool.Dial()
 }
+
+func isSentinelConnection() bool {
+	return sentinelConnection
+}
+
+func validateConnection(conn *RedisConn) bool {
```

Comment:
It is used here: https://github.com/benmanns/goworker/pull/55/files#diff-ac87330cabd803f80c2aa07c8b65c81fR117

By putting this method here it means I can keep some of the sentinel dependencies confined to this file.